src/MongoDB/ServerDescription.c

```diff
@@ -115,7 +125,7 @@ static PHP_METHOD(ServerDescription, getType)
 
 	PHONGO_PARSE_PARAMETERS_NONE();
 
-	RETVAL_LONG(php_phongo_server_description_type(intern->server_description));
+	RETVAL_STRING(mongoc_server_description_type(intern->server_description));
```

Comment:
This would be a good opportunity to revise `serverDescription-getType-001.phpt` similar to what you did for the `TopologyDescription::getType()` test.

Additionally, `php_phongo_serverdescription_get_properties_hash` can be changed to output a "type" as a string instead of calling `php_phongo_server_description_type`. Its test can also be revised to use `%r` like the corresponding TopologyDescription tests.